tests/unit/helpers.test.ts

```diff
@@ -0,0 +1,37 @@
+import { getEtaDate } from '../../lib/helpers';
+
+describe('getEtaDate', () => {
+  it('should return a date string for YYYY-MM-DD', () => {
+    // dates
+    expect(getEtaDate('eta:2022-11-14')).toBe('2022-11-14');
+    expect(getEtaDate('eta : 2022-11-14')).toBe('2022-11-14');
+    expect(getEtaDate('eta: 2022-11-14')).toBe('2022-11-14');
+  });
+
+  it('should return a date string for quarters and optional years', () => {
+    // quarters
+    expect(getEtaDate('eta: q4 2022')).toBe('2022-12-31');
+    expect(getEtaDate('eta: q3 2024')).toBe('2024-09-30');
```

Comment:
this format is invalid. We should use `YYYYQN`

Reply:
fixed.